pkg/management/component.go

```diff
@@ -932,6 +934,50 @@ func (c *component) DeleteCredentialsForUser(ctx context.Context, realmName stri
 	return err
 }
 
+func (c *component) UnlockCredentialForUser(ctx context.Context, realmName string, userID string, credentialID string, credType string) error {
```

Comment:
Do we really need to be able to provide credType as we can determine it by ourself ?